### ISSUE TITLE
Problem uploading worker script

### ISSUE THREAD

**Issue by beddari** [thingalon/pony-ssh#11]
Hi, thank you for creating pony-ssh,
I get an error trying to connect to a remote host running Fedora 31 & Python 3.7. I'm using v0.3.0. Happy to help debug this further.

```
[2020-02-03 10:26:56] [info] Connection established
[2020-02-03 10:26:56] [info] Preparing worker script
[2020-02-03 10:26:57] [warn] STDERR output during worker upload: Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib64/python3.7/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 10: invalid start byte

[2020-02-03 10:26:57] [error] Error connecting to beddari: Error code 1 while uploading worker script. STDERR says: Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib64/python3.7/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 10: invalid start byte
```

**Comment by thingalon** [comment]
Hey @beddari! Thanks for the bug report :)

I've found some issues with python3 compatibility. I've prepared a fix over in https://github.com/thingalon/pony-ssh/pull/12.

I will roll out a fresh version with this fix soon.

**Comment by thingalon** [comment]
Ok, that's shipped in Pony-ssh 0.4.0. Can you please install the latest version and try your connection again?

**Comment by thingalon** [comment]
Closing this. Feel free to reopen it if you still have the same trouble with the latest version.
